charts/argo-cd/Chart.yaml

```
@@ -21,4 +21,4 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Feat]: Add ability to set Notification Services"
+    - "[chore]: update comment to clarify usecase"
```

```suggestion
    - "[Changed]: update comment to clarify usecase"
```
We use these templates here:
https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog

---

done.